app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -30,14 +30,29 @@
             <div v-show="statusFields">
               <div v-if="showAsignee">
                 <!-- {{ formReviewers }} -->
-                <label>Assign To</label>
+                <label>
+                  Assign To
+                  <v-tooltip bottom>
+                    <template #activator="{ on, attrs }">
+                      <v-icon color="primary" v-bind="attrs" v-on="on">
+                        help_outline
+                      </v-icon>
+                    </template>
+                    <span>
+                      Submissions can be assigned to Form Reviewers. <br />
+                      To add more team members as Form Reviewers, go to the
+                      Manage page for this form.
```

Comment:
I wonder if it would be a good idea to make the "Manage page" part of text hyperlinkable?

Reply:
Problem is it's possible that you can be here (you're a reviewer) but don't have access to the Manage page at all...
I think this'll need some iteration after getting users on it but yeah a link would be good, but would have to do the same role detection.